configuration/api_test.go

```diff
@@ -271,12 +274,22 @@ func createMockedResponder(t *testing.T, method string) roundTripFunc {
 			}
 		}
 
-		if req.URL.String() != "https://api.livechatinc.com/v3.4/configuration/action/"+method {
+		expectedMethod := http.MethodPost
+		if method == "get_organization_id" || method == "get_license_id" {
+			expectedMethod = http.MethodGet
+		}
+
+		url := "https://api.livechatinc.com/v3.4/configuration/action/" + method
+		if expectedMethod == http.MethodGet && !strings.Contains(req.URL.String(), url) {
```

Comment:
maybe `HasPrefix` is better suited here, as we need req URL to start with expected URL?

Reply:
Right, `HasPrefix` maybe the better choice.